fractal_server/app/schemas/v2/collection_state.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+from typing import Any
+from typing import Optional
+
+from pydantic import BaseModel
+from pydantic import validator
+
+from fractal_server.app.schemas._validators import valutc
+
+
+class CollectionStateReadV2(BaseModel):
```

Comment:
Shall we move this definition to `fractal_server/app/schemas/v2/task_collection.py`?